compiler/rustc_lint/src/lib.rs

```diff
@@ -303,19 +303,26 @@ fn register_builtins(store: &mut LintStore) {
         BARE_TRAIT_OBJECTS,
         UNUSED_EXTERN_CRATES,
         ELLIPSIS_INCLUSIVE_RANGE_PATTERNS,
-        ELIDED_LIFETIMES_IN_PATHS,
-        EXPLICIT_OUTLIVES_REQUIREMENTS // FIXME(#52665, #47816) not always applicable and not all
-                                       // macros are ready for this yet.
-                                       // UNREACHABLE_PUB,
+        ELIDED_LIFETIMES_IN_PATHS_TIED,
+        ELIDED_LIFETIMES_IN_PATHS_UNTIED,
+        EXPLICIT_OUTLIVES_REQUIREMENTS,
+        // FIXME(#52665, #47816) not always applicable and not all
+        // macros are ready for this yet.
+        // UNREACHABLE_PUB,
+        // FIXME macro crates are not up for this yet, too much
+        // breakage is seen if we try to encourage this lint.
+        // MACRO_USE_EXTERN_CRATE
+    );
 
-                                       // FIXME macro crates are not up for this yet, too much
-                                       // breakage is seen if we try to encourage this lint.
-                                       // MACRO_USE_EXTERN_CRATE
+    add_lint_group!(
+        "elided_lifetimes_in_paths",
+        ELIDED_LIFETIMES_IN_PATHS_TIED,
+        ELIDED_LIFETIMES_IN_PATHS_UNTIED,
     );
 
     // Register renamed and removed lints.
     store.register_renamed("single_use_lifetime", "single_use_lifetimes");
-    store.register_renamed("elided_lifetime_in_path", "elided_lifetimes_in_paths");
+    // store.register_renamed("elided_lifetime_in_path", "elided_lifetimes_in_paths");
```

Comment:
Why is this being commented out? Is it not compatible with having a lint group as the second actual argument?

Reply:
Correct. I've [asked about that](https://rust-lang.zulipchat.com/#narrow/stream/147480-t-compiler.2Fwg-diagnostics/topic/renaming.20to.20a.20lint.20group).

Reply:
I added some code to handle this kind of renaming.